modules/ebi-metagenomics/summarisegoslims/meta.yml

```diff
@@ -0,0 +1,75 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "summarisegoslims"
+description: Script that generates counts from an InterProScan output and a GO Annotation File (GAF) for both GO terms and GO-Slim terms.
+keywords:
+  - GO
+  - GO-slim
+  - interpro
+  - interproscan
+  - GAF
+  - owltools
+tools:
+  - "summarisegoslims":
+      description: "Script that summarises GO terms into GO-Slim terms, uses mgnify-pipelines-toolkit as conda environment"
+      homepage: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      documentation: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      tool_dev_url: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      licence: ["Apache-2.0"]
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - ips:
+        type: file
+        description: InterProScan output tsv file.
+        pattern: "*.tsv"
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - gaf:
+        type: file
+        description: GAF file containing the GO-slim definitions, made by Owltools.
+        pattern: "*.gaf"
+  - - go_obo:
+        type: file
+        description: Gene ontology .OBO file, for example go-basic. Can usually be downloaded here https://geneontology.org/docs/download-ontology/
+        pattern: "*.go"
+  - - go_banding:
+        type: file
+        description: txt file containing the GO terms and descriptions that make up the input GO-slim
+        pattern: "*.txt"
+output:
+  - go_summary:
+      - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*_summary":
+          type: file
+          description: Textfile containing the counts for the different GO terms in the InterProScan output.
+          pattern: "*_summary"
```

Comment:
The summaries are CSV like files, right? ( https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit/pull/28/files ). Can you add the extension please?

Reply:
done!